### DISCUSSION TITLE
.net 7

### DISCUSSION THREAD

**Discussion by AsertCreator** [CosmosOS/Cosmos#2497] [General]
im cringing rn

**Comment by AsertCreator** [comment]
hmm.. i think net 7 doesnt have so much features to implement. net 6 doesnt have either. c#10 & c#11 have only new compile-time features.

**Reply by jspa2** [reply]
Native AOT is a big one but that'll be a lot of effort to add to Cosmos

**Reply by zarlo** [reply]
> Native AOT is a big one but that'll be a lot of effort to add to Cosmos

we did look at using Native AOT bit at this time we will not likely be using it

**Comment by AsertCreator** [comment]
also, do records work on cosmos?

**Comment by zarlo** [comment]
we will not be supporting non LTS versions as updating .net versions can take a ver long time so the next up date will be for .net 8

**Reply by AsertCreator** [reply]
oh okay
